docs/software/osg-build-tools.md

```diff
@@ -4,125 +4,234 @@ OSG Build Tools
 
 This page documents the tools used for RPM development for the OSG Software Stack. See [the RPM development guide](../software/rpm-development-guide.md) for the principles on which these tools are based.
 
-The tools are distributed in the `osg-build` RPM in our repositories, but can also be used from a Git clone of [opensciencegrid/osg-build on GitHub](https://github.com/opensciencegrid/osg-build).
+The tools are available in Git in [opensciencegrid/osg-build on GitHub](https://github.com/opensciencegrid/osg-build).
```

Comment:
Doesn't have to be solved here but could we just change this to `osg-htc/osg-build`? I'm thinking that we update the description in the quarter-jar org pointing to the new repo and archive the old one

Reply:
Should be safe just to move the repo and let GitHub handle the redirection.

Reply:
Oh good point! I forgot that it's possible to move repos [between orgs](https://docs.github.com/en/repositories/creating-and-managing-repositories/transferring-a-repository#transferring-a-repository-owned-by-your-organization)